tree-sitter-dtd/grammar.js

```diff
@@ -12,6 +12,11 @@ const O = optional;
 module.exports = grammar({
   name: 'dtd',
 
+  externals: $ => [
+    $._non_xml_name,
```

Comment:
There's no need to keep the alias if you can name it like this.

```suggestion
    $.PiTarget,
```